rustls-mbedpki-provider/src/lib.rs

```diff
@@ -20,6 +20,21 @@ pub mod server_cert_verifier;
 pub use client_cert_verifier::MbedTlsClientCertVerifier;
 pub use server_cert_verifier::MbedTlsServerCertVerifier;
 
+/// A config about whether to check certificate validity period
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct CertActiveCheck {
+    /// If accept expired certificate, default to false
```

Comment:
Maybe could be "Accept expired certificates"?

Reply:
done